### ISSUE TITLE
Feature Request: Command Palette (Like Atom, Sublimetext, or Evernote)

### ISSUE THREAD

**Issue by incorvia** [wavebox/waveboxapp#244]
It would be great to have a keyboard command that brings up a command palette with quick jump access to all of your accounts.

**Comment by Thomas101** [comment]
Yeah that's a great idea! Have you also tried using the <kbd>cmd</kbd>+<kbd>1</kbd>, <kbd>cmd</kbd>+<kbd>2</kbd>etc shortcuts to switch between accounts?

**Comment by incorvia** [comment]
I'm glad you agree! I have used those. But sometimes I want to jump from
one account's gmail to another accounts drive.. and I have to use two
different keyboard commands. Be nice if I could just jump straight there.



- Kevin

**Comment by mathieutu** [comment]
Hi!
Some news about that @Thomas101. It can be a real game changer actually. 
[Station](https://getstation.com/) is doing that, and it's amazing!

Can search in every account BUT also inside them:
<img src="https://getstation.com/public/images/feature-2-v2@2x.png" alt="Screenshot - Search">

Examples: from calendar searching directly my name:
<img src="https://goo.gl/8cdF3p">

Searching "staffing" links directly to staffing channel in slack, and staffing spreadsheet in drive.
<img src="https://goo.gl/uKAaqE">


1st part is the previously/regularly opened, and then all the choices.

I have no idea how to do that, but I'm convinced it will be a real real true improvement for wavebox!

**Comment by Thomas101** [comment]
We're working on a few tools to make switching & managing accounts easier in this beta cycle. Over the past couple of releases we've added support for recents, pinned etc. 4.5.7 that just shipped there's support for <kbd>Ctrl</kbd>+<kbd>Tab</kbd> for quickly switching accounts
![b](https://user-images.githubusercontent.com/103586/50152567-d360ff00-02bb-11e9-98d4-9001d0803b1e.png)

and more to come... :-)

**Comment by mathieutu** [comment]
Ok, thanks.
I've seen that and it's cool, but it will not replace jumping like above.

(Moreover, quick switching works only between awaken main services (and not sleeping and sub ones))

Do you think we could anything to help you on that feature?

**Comment by Thomas101** [comment]
> and more to come... :-)

Working on it :)

**Comment by mathieutu** [comment]
Hi, I've tried it. Thanks, it's really cool, and a very good start.

The recent searches/found at start when nothing is typed would be a cool thing.

However, this feature in not as good as Station one yet: a true way to search in channels/people/emails/calendar events/docs will be **just awesome**.

😉

**Comment by TheSpyder** [comment]
I started using this today, and I agree pre-populating the command palette with my accounts would be really handy - a bit like ctrl+tab, which I don't use because ctrl+anything is a pain on mac laptop keyboards :)

**Comment by Thomas101** [comment]
4.5.14 prepopulates quick-switch. There's more to be done here though!

**Comment by Thomas101** [comment]
This has just been released in the latest version of Wavebox (4.7.0). As I mentioned above, there's still work to do on this and we're hoping to add more in during the next beta cycle :). Thanks!

**Comment by rawkode** [comment]
@Thomas101 How is this issue looking? :D

**Comment by rawkode** [comment]
Sorry @Thomas101, I missed the Control-T comment. Looking good!
